### DISCUSSION TITLE
Keep reactions unlocked when an issue is stale/locked

### DISCUSSION THREAD

**Discussion by charlieforward9** [community/community#51556] [Projects and Issues] [answered]
There are multiple occasions where an issue will go stale/locked and a solution is present within the conversation but I am unable to "boost" it because is read-only. This feature will keep old issues "reactively-alive", which could help developers reach solutions faster. Furthermore, it seems to be a minor modification.

Thanks for making great software, GitHub Team!

**Comment by roke-julian-lockhart** [comment, marked as answer]
@charlieforward9, is this a duplicate of [`discussions/36107`](https://github.com/orgs/community/discussions/36107#discussion-4472711)?
